lib/oli_web/live/curriculum/rollup.ex

```diff
@@ -44,6 +44,7 @@ defmodule OliWeb.Curriculum.Rollup do
 
           partial_activity_map =
             resolved_activities
+            |> Enum.filter(fn r -> !is_nil(r) end)
```

Comment:
May apply to above - a small private function could save repetition here and make it easier to change in the future.